From 616d48f5f9030a5295f04caa9f3685259a2742b2 Mon Sep 17 00:00:00 2001
From: "Yan, WalterX" <walterx.yan@intel.com>
Date: Wed, 30 Jan 2019 18:20:07 +0800
Subject: [PATCH] Enable the mAddUserView after adding a user.

mAddUserView is disabled in adding a user,
so enable it after adding done.

Change-Id: I5b32c97d9f03c4030ef53890939da5088415bcd3
Tracked-On: OAM-75529
Signed-off-by: Yan, WalterX <walterx.yan@intel.com>
---
 .../src/com/android/systemui/statusbar/car/UserGridRecyclerView.java   | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/packages/SystemUI/src/com/android/systemui/statusbar/car/UserGridRecyclerView.java b/packages/SystemUI/src/com/android/systemui/statusbar/car/UserGridRecyclerView.java
index a49d507..31a2d03 100644
--- a/packages/SystemUI/src/com/android/systemui/statusbar/car/UserGridRecyclerView.java
+++ b/packages/SystemUI/src/com/android/systemui/statusbar/car/UserGridRecyclerView.java
@@ -294,6 +294,9 @@ public class UserGridRecyclerView extends PagedListView implements
 
             @Override
             protected void onPostExecute(UserInfo user) {
+                if (mAddUserView != null) {
+                    mAddUserView.setEnabled(true);
+                }
                 if (user != null) {
                     mUserManagerHelper.switchToUser(user);
                 }
-- 
1.9.1

